addons/purchase/views/account_move_views.xml

```diff
@@ -9,14 +9,19 @@
             <!-- Auto-complete could be done from either a bill either a purchase order -->
             <field name="invoice_vendor_bill_id" position="after">
                 <field name="purchase_id" invisible="1"/>
-                <field name="purchase_vendor_bill_id"
+                <label for="invoice_vendor_bill_id" string="Auto-Complete" class="oe_edit_only"
+                       attrs="{'invisible': ['|', '|', ('state','not in',['draft']), ('state', '=', 'purchase'), ('move_type', '!=', 'in_invoice')]}" />
```

Comment:
that's pretty much tordu: \<label for a X field with a string and followed by a field which isn't the X-field>

what's the reason of all this brol?

there has to be a simpler way...

Reply:
I know, but if I put class="oe_edit_only" on the field without doing this it doesn't work, I could figure out why.